### PR TITLE
Add CS2101 version of the UG

### PR DESCRIPTION
**Pull request by ketamethane**
Update CS2103T UG to address marking a marked question, and unmarking an unmarked question.

As per @qiaoen17 's notes:
Note that not all features are explained (nicely)

Front matters:

- [ ] add more picture
- [ ] friendly tone
- [x] explain how to use CLI
- [x] add in purpose of the guide e.g. the purpose of the guide is blahblah
- [ ] make everything clear and explicit 

- [x] Group features according to students, tutorials and questions 
- [x] Do a preamble, this app has 3 main features: student, tutorial, question
- [ ] why they need to do what
- [x] After student, tutorial and question title write what the feature is: 

Student
- [x] Add students details in, edit student details, add help tag  

Tutorial
- [x] Add tutorial details in, mark tutorial as complete

Question
- [x] Add question details, mark questions as important

- [ ] In order to make sure your command is properly executed, please note the following 
- [x] Command format should not be under features 

- [x] State purpose of the features

- [ ] Lack of navigation to allow the readers 

- [x] Command summary: what commands are used at one glance
- [ ] if you want to have a quick glance/ play with commands here you go

- [ ] Need to show reader consideration 

- [ ] Can add comments like well done or successful

### CODE REVIEW

File: docs/UserGuideCS2101.md [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+questions asked by students. SETA is optimized for use via a Command Line Interface (CLI) while still having the
+benefits of a Graphical User Interface (GUI). If you can type fast, SETA enables you to track your students, manage your
+tutorials and note down questions more effectively than traditional GUI apps.
+
+The purpose of this guide is to inform you what features are available in SETA, and how to use them to solve your 
+needs as a CS2103T TA.
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: What is a Command Line Interface (CLI)?**<br>
```
Comment (qiaoen17) [inline docs/UserGuideCS2101.md:13]: May want to include what icons mean here if we plan to use icons at the start of the user guide :)

File: docs/UserGuideCS2101.md [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+
+It is a programme that allows you to create, modify and delete files using a text-based user interface. In SETA, all 
+you have to do is to type in the command box and press enter to track your students and manage your questions and
```
Comment (qiaoen17) [inline docs/UserGuideCS2101.md:16]: Probably need to show what a command box is, probably give a photo here and like draw a red rectangle around the command box. Perhaps User interface can change to display instead?

File: docs/UserGuideCS2101.md [hunk 3]
```diff
@@ -0,0 +1,424 @@
...
+            * Marking a tutorial: [`marktut`](#marking-a-tutorial-marktut)
+        * **Exiting the program**: [`exit`](#exiting-the-program--exit)
+    * **[FAQ](#faq)**
+    * **[Command Summary](#command-summary)**
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Quick Start
+
+1. Ensure you have Java `11` or above installed in your Computer.
```
Comment (qiaoen17) [inline docs/UserGuideCS2101.md:50]: to add hyperlink to install Java 11, and we can add a link to help page from Java website for users facing trouble with installing Java 11?

File: docs/UserGuideCS2101.md [hunk 4]
```diff
@@ -0,0 +1,424 @@
...
+
+1. Download the latest `SETA.jar` from [here](https://github.com/AY2223S1-CS2103T-T08-4/tp/releases).
+
+1. Copy the file to the folder you want to use as the _home folder_ for your SETA.
+
+1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app
+   contains some sample data.<br>
+   ![Ui](images/Ui.png)
```
Comment (qiaoen17) [inline docs/UserGuideCS2101.md:58]: To update Ui image to match our current startup Ui image 👍

File: docs/UserGuideCS2101.md [hunk 5]
```diff
@@ -0,0 +1,424 @@
...
+
+* **`exit`** : Exits the app.
+
+Refer to the [Features](#features) below for details of each command.
+
+--------------------------------------------------------------------------------------------------------------------
+
+## Features
+
+SETA contains features that allow you to manage 3 kinds of lists:
```
Comment (qiaoen17) [inline docs/UserGuideCS2101.md:78]: Phrasing here could be 3 things instead of 3 kinds of lists? 😓

File: docs/UserGuideCS2101.md [hunk 6]
```diff
@@ -0,0 +1,424 @@
...
+
+* Parameters can be in any order.<br>
+  e.g. if the command specifies `n/NAME e/EMAIL`, `e/EMAIL n/NAME` is also acceptable.
+
+* If a parameter is expected only once in the command, but you specified it multiple times, only the last occurrence of
+  the parameter will be taken.<br>
+  e.g. if you specify `h/@johnlim h/@johnlimjj`, only `h/@johnlimjj` will be taken.
+
+* Extraneous parameters for commands that do not take in parameters (such as `liststu` and `exit`) will be ignored.<br>
+  e.g. if the command specifies `liststu 123`, it will be interpreted as `liststu`.
```
Comment (qiaoen17) [inline docs/UserGuideCS2101.md:103]: Good job for this :)

File: docs/UserGuideCS2101.md [hunk 7]
```diff
@@ -0,0 +1,424 @@
...
+Sometimes, you may receive questions that you may not have the answer to or you may not have enough time to answer 
+all of them during the tutorial. SETA allows you to track these questions. Furthermore, if you find a question that 
+requires urgent clarification or critical thinking, you can indicate on the user interface that these questions are 
+important. For questions requiring critical thinking, it would be also good to share them with other TAs so that the 
+rest of the module's students can benefit from knowing this question.
+
+### Adding a question : `addq`
+
+Adds a question to the question list.
+This feature allows you to keep track of the students' questions.
```
Comment (qiaoen17) [inline docs/UserGuideCS2101.md:256]: Of your student's questions instead of the students' questions

File: docs/UserGuideCS2101.md [hunk 8]
```diff
@@ -0,0 +1,424 @@
...
+* The index must be within the number of questions in the question list. E.g. There are 4 questions. The possible
+  indexes are 1, 2, 3 and 4.
+
+Example:
+
+* `markq 1` marks the first question in the question list as important
+
+### Unmarking a question : `unmarkq`
+
+Marks a question as unimportant. (If the question was previously or mistakenly marked as important)
```
Comment (qiaoen17) [inline docs/UserGuideCS2101.md:284]: Allows you to mark a question as unimportant if the question was previously or mistakenly marked as important

File: docs/UserGuideCS2101.md [hunk 9]
```diff
@@ -0,0 +1,424 @@
...
+* The index must be a positive integer 1, 2, 3, ...
+* The index must be within the number of questions in the question list. E.g. There are 4 questions. The possible
+  indexes are 1, 2, 3 and 4.
+
+Example:
+
+* `deleteq 1` deletes the first question from the question list
+
+### Tutorials
+Tutorials contain a `group number`, `content` to be taught, and its `date and time`.
```
Comment (qiaoen17) [inline docs/UserGuideCS2101.md:315]: Possible phrasing instead could be: Each tutorial has a group number, a topic to focus on and a date and time. These correspond to the `GROUP_NUMBER`, `CONTENT` and `DATE TIME` parameters accordingly.

File: docs/UserGuideCS2101.md [hunk 10]
```diff
@@ -0,0 +1,424 @@
...
+* The index must be a positive integer 1, 2. 3, … .
+*
+
+Example:
+
+* `deletetut 1`deletes the first tutorial from the tutorial list
+
+### Marking a tutorial: `marktut`
+
+Marks content in the tutorial as done.
```
Comment (qiaoen17) [inline docs/UserGuideCS2101.md:348]: Marks the tutorial as done, not just the content.

File: docs/UserGuideCS2101.md [hunk 11]
```diff
@@ -0,0 +1,424 @@
...
+SETA data are saved as a JSON file `[JAR file location]/data/seta.json`. Advanced users are welcome to update data
+directly by editing that data file.
+
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+If your changes to the data file makes its format invalid, SETA will discard all data and start with an empty data file at the next run.
+</div>
+
+### Archiving data files `[coming in v2.0]`
+
+_Details coming soon ..._
```
Comment (qiaoen17) [inline docs/UserGuideCS2101.md:398]: might just want to delete this

File: docs/UserGuideCS2101.md [hunk 12]
```diff
@@ -0,0 +1,424 @@
...
+
+
```
Comment (qiaoen17) [inline docs/UserGuideCS2101.md:400]: Add the clear feature here too because we have it in still!

**Review by qiaoen17 — COMMENTED** [review]
Thank you for helping add more you language 👍

File: docs/UserGuideCS2101.md (hunk 2 above)
Comment (ketamethane) [reply docs/UserGuideCS2101.md:16]: Ok, I'll rectify this

File: docs/UserGuideCS2101.md (hunk 3 above)
Comment (ketamethane) [reply docs/UserGuideCS2101.md:50]: added

File: docs/UserGuideCS2101.md (hunk 4 above)
Comment (ketamethane) [reply docs/UserGuideCS2101.md:58]: ok added

File: docs/UserGuideCS2101.md (hunk 5 above)
Comment (ketamethane) [reply docs/UserGuideCS2101.md:78]: done

File: docs/UserGuideCS2101.md (hunk 8 above)
Comment (ketamethane) [reply docs/UserGuideCS2101.md:284]: done

File: docs/UserGuideCS2101.md (hunk 9 above)
Comment (ketamethane) [reply docs/UserGuideCS2101.md:315]: changed it

File: docs/UserGuideCS2101.md (hunk 10 above)
Comment (ketamethane) [reply docs/UserGuideCS2101.md:348]: done

File: docs/UserGuideCS2101.md (hunk 11 above)
Comment (ketamethane) [reply docs/UserGuideCS2101.md:398]: done

File: docs/UserGuideCS2101.md (hunk 12 above)
Comment (ketamethane) [reply docs/UserGuideCS2101.md:400]: done

File: docs/UserGuideCS2101.md (hunk 1 above)
Comment (ketamethane) [reply docs/UserGuideCS2101.md:13]: Maybe I can leave it to the rest? I'm still not sure how we can improve the reader friendliness.
There's also the navigation issue that I have not resolved yet.

**Pull request merged**